label_studio_ml/examples/segment_anything_model/start.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 # Execute the gunicorn command
-exec gunicorn --preload --bind :${PORT:-9090} --workers 1 --threads 8 --timeout 0 _wsgi:app
+exec gunicorn --bind :${PORT:-9090} --workers 1 --threads 8 --timeout 0 _wsgi:app
```

Comment:
Why was `--preload` removed?

Reply:
https://stackoverflow.com/questions/72779926/gunicorn-cuda-cannot-re-initialize-cuda-in-forked-subprocess
https://github.com/pytorch/pytorch/issues/116543